backend/server/rhnServer/server_suse.py

```diff
@@ -19,17 +19,13 @@ class SuseData:
   def __init__(self):
     log_debug(4, "SuseData initialized")
     # format:
-    # suse_products { 'guid' = ...
-    #                 'secret' = ...
-    #                 'ostarget' = ...
-    #                 'products' = [{ 'name' = ..., 'version' = ..., 'release' = ..., 'arch' = ..., 'baseproduct' = ...},
-    #                               ...]
-    #               }
+    # suse_products [{ 'name' = ..., 'version' = ..., 'release' = ..., 'arch' = ..., 'baseproduct' = ...},
+    #               ...]
     self.suse_products = {}
```

Comment:
Since `self.suse_products` is expected to be a `list` now, maybe we should also:

```suggestion
    self.suse_products = []
```